font-codegen/src/table.rs

```diff
@@ -190,15 +190,15 @@ pub(crate) fn generate_group(item: &GenericGroup) -> syn::Result<TokenStream> {
 
     let mut variant_decls = Vec::new();
     let mut read_match_arms = Vec::new();
-    let mut as_some_table_arms = Vec::new();
+    let mut dyn_inner_arms = Vec::new();
```

Comment:
Nit: Feels like Hungarian notation which isn't really necessary in Rust. Would anything be particularly horrible about dropping dyn_ from various names? - we know it's dyn, it's in the signature.

Reply:
nothing terrible, but the change here is just making this small portion of the code match the naming convention used everywhere else. This method is a private implementation detail related to the traversal code, and I don't have particularly strong feelings either way (the other name I had used here was 'as_some_table')

in any case if we want to change this it would be a significantly bigger diff and should be a separate PR.